scripts/ci/deploy.sh

```diff
@@ -48,8 +48,8 @@ if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
   ./scripts/build/docker/run-command.sh \
     -r "$ARGV_ARCHITECTURE" \
     -s "$(pwd)" \
-    -c "make publish-aws-s3"
+    -c "make publish-aws-s3 publish-bintray-debian publish-bintray-redhat"
 else
   ./scripts/build/check-dependency.sh make
-  make publish-aws-s3
+  make publish-aws-s3 publish-bintray-debian publish-bintray-redhat
```

Comment:
I'm not sure about this change - it means that OSX will be trying to make `publish-bintray-debian` ?
What if we instead created a new `deploy` phoney target in the Makefile, which the Makefile could then conditionally add any of the `publish-aws-s3`, `publish-bintray-debian`, `publish-bintray-redhat` targets to as necessary, and then this script would just do `make deploy`?